src/shared/bpf-dlopen.c

```diff
@@ -100,7 +101,8 @@ int dlopen_bpf(void) {
                                 DLSYM_ARG_FORCE(libbpf_probe_bpf_prog_type)
 #endif
                 );
-        }
+        if (r < 0)
+                return log_debug_errno(r, "Failed to load symbols from libbpf: %m");
```

Comment:
no need for a log msg here. the functions already logged at LOG_DEBUG level, you even specified the intended log level